core/src/main/java/org/jruby/RubyArray.java

```diff
@@ -4100,7 +4077,9 @@ public IRubyObject sample(ThreadContext context, IRubyObject[] args) {
             if (args.length > 0) {
                 IRubyObject hash = TypeConverter.checkHashType(context.runtime, args[args.length - 1]);
                 if (!hash.isNil()) {
-                    IRubyObject[] rets = ArgsUtil.extractKeywordArgs(context, (RubyHash) hash, new String[] { "random" });
+                    IRubyObject[] rets = ArgsUtil.extractKeywordArgs(context, (RubyHash) hash,
```

Comment:
let's keep it consistently a one liner maybe :) ?

Reply:
@kares sure :)